dev/RadioButtons/ColumnMajorUniformToLargestGridLayout.cpp

```diff
@@ -135,8 +135,7 @@ void ColumnMajorUniformToLargestGridLayout::OnRowSpacingPropertyChanged(const wi
 
 void ColumnMajorUniformToLargestGridLayout::OnMaximumColumnsPropertyChanged(const winrt::DependencyPropertyChangedEventArgs& args)
 {
-    auto const maxColumns = MaximumColumns();
-    if (maxColumns <= 0)
+    if (args.NewValue() <= 0)
```

Comment:
is NewValue() a double or a winrt::IInspectable? I would have expected an unbox_double here.

Reply:
:( you are right. fixing